### ISSUE TITLE
See if we actually reached out to brandon once a week

### ISSUE THREAD

**Issue by fzanzucc** [asu-cis-capstone/schoolfly#17]

**Comment by bclarke25** [comment]
hey guys... yeah, haven't heard anything in a while.  seems like we should have more comms.  what's the latest?

**Comment by tjlindahl** [comment]
Yeah, we discussed today that we should be meeting more or at least communicating with you more than we have been. 
     Anyways, we have a pretty basic version of what we expect our flyer pages to be like, with the ability to add, edit, and delete flyers. There is no user authentication yet, so anyone can add, edit, or delete any post, but we are working on user authentication.
     Also, we will be working in a home page for the site.

     Sorry about the lack of communication, we plan on changing that from now on.

**Comment by bclarke25** [comment]
Cool. Just glad you guys have made progress. Let me know when I can check
it out. Thx for the update.

--

Brandon
